package.json

```diff
@@ -9,9 +9,10 @@
     "compile": "webpack",
     "karma": "karma start karma.config.js",
     "lint": "eslint '**/*.js'",
+    "lint-fix": "eslint '**/*.js' --fix",
     "prepublish": "yarn compile",
     "start": "webpack-dev-server",
-    "test": "yarn ci:lint && yarn ci:test",
+    "test": "yarn ci:test && yarn ci:lint",
```

Comment:
I'm saying let's do `yarn lint-fix && yarn ci:lint && yarn ci:test`. We still keep the order, but fix the stupid stuff first.